lib/Importer/ONNXIFIModelLoader.cpp

```diff
@@ -59,6 +61,30 @@ llvm::Expected<std::unique_ptr<ONNXIFIModelLoader>> ONNXIFIModelLoader::parse(
 
     // Keep hold of the context
     loader->core_ = std::move(onnxLoader);
+  } else {
+    // Use Caffe2 Model loader
+    std::unique_ptr<Caffe2ModelLoader> c2Loader(
+        new Caffe2ModelLoader(F, &loaderConstructionErr));
+    if (loaderConstructionErr) {
+      return std::move(loaderConstructionErr);
+    }
+
+    caffe2::NetDef networkDef;
+    ASSIGN_VALUE_OR_RETURN_ERR(networkDef,
+                               c2Loader->loadProto(model, modelSize));
+
+    RETURN_IF_ERR(c2Loader->loadWeights(weightsCount, weightDescriptors));
```

Comment:
I'm confused, shouldn't this be `c2Loader->loadWeightsFromNet` why does this compile?

Reply:
`loadWeightsFromNet` is a different function where we load the weights from the init net. Here we load weights from the TensorDescriptor.